docs/source/sampling.rst

```diff
@@ -10,9 +10,9 @@ The package provides functions for sampling from a given population (with or wit
 
     Randomly draw an element from an array ``a``.
 
-.. function:: sample(a, n[; replace=true, ordered=false])  
+.. function:: sample(a, n[; replace=true, ordered=false])
 
-    Randomly draw ``n`` elements from ``a``. 
+    Randomly draw ``n`` elements from ``a``.
```

Comment:
That's fine, a lot of editors do that. Makes the diff marginally harder to review, but I think it's worth having cleaner whitespace.